mlpipeline/mlpipeline/ingredient_parser/parser.py

```
@@ -1,60 +1,82 @@
+    spaced_text = re.sub(unicode_fraction_pattern, ' ', fraction_str)
+    normalized = unicodedata.normalize('NFKC', spaced_text)
+    
+    normalized = normalized.replace('\u2044', '/')
+    try:
+        if normalized[0] == '-':
+            return -float(sum(Fraction(part) for part in normalized[1:].split()))
+        
+        return float(sum(Fraction(part) for part in normalized.split()))
+    except ValueError as err:
+        print("Error converting float to fraction in text:")
+        raise err
```

Ruff (0.14.10)

Use `raise` without specifying exception name

Remove exception name

(TRY201)